AdyenDropIn/DropInComponent.swift

```diff
@@ -158,7 +158,8 @@ public final class DropInComponent: NSObject, PresentableComponent {
     }
     
     internal lazy var rootComponent: PresentableComponent = {
-        if let preselectedComponents = componentManager.storedComponents.first {
+        if configuration.allowPreselectedPaymentView,
+           let preselectedComponents = componentManager.storedComponents.first {
```

Comment:
but then if `configuration.allowPreselectedPaymentView` is false, you need to show all stored payment methods in the second payment method list, right?

Reply:
Of cause, all stored payment methods are there:
This only allows to skip `PreselectedPaymentMethodComponent` and goes directly to a `PaymentMethodListComponent`.